packages/api-server/src/plugins/withFunctions.ts

```diff
@@ -66,42 +67,45 @@ export const setLambdaFunctions = async (foundFunctions: string[]) => {
   })
 }
 
-const lambdaRequestHandler = async (req: Request, res: Response) => {
+interface LambdaHandlerRequest extends RequestGenericInterface {
+  Params: {
+    routeName: string
+  }
+}
+
+// This will take a express request
+// Then convert it to a lambdaEvent, and pass it to the the approrpiate hanlder for the routeName
+// The LAMBDA_FUNCTIONS lookup has been populated already by this point
+const lambdaRequestHandler = async (
+  req: FastifyRequest<LambdaHandlerRequest>,
+  reply: FastifyReply
+) => {
   const { routeName } = req.params
+
   if (!LAMBDA_FUNCTIONS[routeName]) {
     const errorMessage = `Function "${routeName}" was not found.`
     console.error(errorMessage)
```

Comment:
Sine `req` has the logger, use `req.log.error` instead?

Pass req/log to building lambdas to Log?